### DISCUSSION TITLE
Directus url in nuxt.config.ts inside public or not?

### DISCUSSION THREAD

**Discussion by notflip** [Intevel/nuxt-directus#227] [Q&A] [answered]
In the docs the folllowing is written, as you can see the directus object is inside the runtime config, however I'm getting a 404 error with this. When I move the directus object inside the public object, then it works.

Is this an issue in the docs or do I have to change something?

In the docs
```
export default defineNuxtConfig({
  modules: ['nuxt-directus'],
  runtimeConfig: {
    directus: {
      url: ""
    }
  }
})
```

My version
```
runtimeConfig: {
    public: {
      directus: {
        url: process.env.DIRECTUS_URL,
      },
      siteUrl: process.env.NUXT_PUBLIC_SITE_URL,
    },
  },
```

**Comment by sandros94** [comment, marked as answer]
Indeed, it should go inside `runtimeConfig.public.directus.url`.
